upsidedown/templates/index.html

```diff
@@ -0,0 +1,57 @@
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+
+<!-- wp:group {"style":{"spacing":{"blockGap":"0px","padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"margin":{"top":"0px","bottom":"0px"}}},"layout":{"type":"constrained","contentSize":"1200px"}} -->
+<div class="wp-block-group" style="margin-top:0px;margin-bottom:0px;padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px">
+    <!-- wp:query {"queryId":0,"query":{"perPage":"6","pages":0,"offset":"0","postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":false},"tagName":"main","displayLayout":{"type":"list"},"layout":{"type":"default"}} -->
```

Comment:
Per this comment pNEWy-fFV-p2#comment-55257, don't we always want to inherit the global query for `index` / home page templates? 

```suggestion
    <!-- wp:query {"queryId":0,"query":{"perPage":"6","pages":0,"offset":"0","postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true},"tagName":"main","displayLayout":{"type":"list"},"layout":{"type":"default"}} -->
```

Otherwise we end up with an ugly URL pattern in pagination: 

<img width="1624" alt="Screenshot 2022-12-07 at 10 19 38 AM" src="https://user-images.githubusercontent.com/5375500/206218465-0f65ef04-e192-4f75-9dc6-2195756f5848.png">

Reply:
Yes I think that's right, thank you!